### PR TITLE
refactor: icon to icons for toasts component

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This component migrates the icons for the toast component to the new icons.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2021-07-07 at 3 06 05 PM

### TESTING INSTRUCTIONS
Manually pulling the branch to hack the toast to present would be the easiest to test as hacking error in ephemereal envior nment is much harder.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/messageToasts/components/Toast.tsx [hunk 1]
```diff
@@ -76,16 +77,17 @@ export default function Toast({ toast, onCloseToast }: ToastPresenterProps) {
     };
   }, [handleClosePress, toast.duration]);
 
-  let iconName: IconName = 'circle-check-solid';
   let className = 'toast--success';
+  let icon = <Icons.CircleCheckSolid css={theme => StyledIcon(theme)} />;
+
   if (toast.toastType === ToastType.WARNING) {
-    iconName = 'warning-solid';
+    icon = <Icons.WarningSolid css={theme => StyledIcon(theme)} />;
```
Comment (suddjian) [inline superset-frontend/src/messageToasts/components/Toast.tsx:84]: This should work I think
```suggestion
    icon = <Icons.WarningSolid css={StyledIcon} />;
```

File: superset-frontend/src/messageToasts/components/Toast.tsx [hunk 2]
```diff
@@ -76,16 +77,17 @@ export default function Toast({ toast, onCloseToast }: ToastPresenterProps) {
...
   let className = 'toast--success';
+  let icon = <Icons.CircleCheckSolid css={theme => StyledIcon(theme)} />;
+
   if (toast.toastType === ToastType.WARNING) {
-    iconName = 'warning-solid';
+    icon = <Icons.WarningSolid css={StyledIcon} />;
     className = 'toast--warning';
   } else if (toast.toastType === ToastType.DANGER) {
-    iconName = 'error-solid';
+    icon = <Icons.ErrorSolid css={theme => StyledIcon(theme)} />;
```
Comment (michael-s-molina) [inline superset-frontend/src/messageToasts/components/Toast.tsx:87]:
```suggestion
    icon = <Icons.ErrorSolid css={StyledIcon} />;
```

File: superset-frontend/src/messageToasts/components/Toast.tsx [hunk 3]
```diff
@@ -76,16 +77,17 @@ export default function Toast({ toast, onCloseToast }: ToastPresenterProps) {
...
     className = 'toast--danger';
   } else if (toast.toastType === ToastType.INFO) {
-    iconName = 'info-solid';
+    icon = <Icons.InfoSolid css={theme => StyledIcon(theme)} />;
```
Comment (michael-s-molina) [inline superset-frontend/src/messageToasts/components/Toast.tsx:90]:
```suggestion
    icon = <Icons.InfoSolid css={StyledIcon} />;
```

**Review by michael-s-molina — APPROVED** [review]

**Pull request merged**